docs/EN_US/ContainerizedHPCC/ContainerizedMods/PVCStorage.xml

```diff
@@ -18,19 +18,19 @@
 
     <orderedlist>
       <listitem>
-        <para>Create data directories using a terminal interface: </para>
+        <para>Create data directories using a terminal interface:</para>
 
         <para>For Windows, use this command:</para>
 
         <para><programlisting>mkdir c:\hpccdata
-mkdir c:\hpccdata\dlls
-mkdir c:\hpccdata\data
-mkdir c:\hpccdata\dali
-</programlisting></para>
+mkdir c:\hpccdata\dalistorage
+mkdir c:\hpccdata\queries
+mkdir c:\hpccdata\sasha
+mkdir c:\hpccdata\hpcc-data</programlisting></para>
```

Comment:
there was a newline before </programlisting> before, is this change ok/deliberate?

Reply:
not deliberate, but innocuous